bitnami/matomo/values.yaml

```diff
@@ -182,6 +182,9 @@ initContainers: []
 ## @param sidecars Attach additional containers to the pod (evaluated as a template)
 ##
 sidecars: []
+## @param serviceAccount Attach serviceAccount to the pod and sidecars
```

Comment:
```suggestion
## @param serviceAccount Attach serviceAccountName to the pod and sidecars
```

Reply:
Thanks @fmulero I figured both mention of `serviceAccount` should get updated